src/pybind/Makefile

```diff
@@ -37,8 +37,9 @@ endif
 CCFILES = \
 chain/chain_pybind.cc \
 chain/chain_supervision_pybind.cc \
-feat/wave_reader_pybind.cc \
 feat/feat_pybind.cc \
+feat/wave_reader_pybind.cc \
+feat/feature_pybind.cc \
```

Comment:
I think `CCFILES` should be sorted alphabetically. That is why
I split them to one cc file per line.

In vim, you can sort them easily. Just visual select the lines to be sorted and
run the command `:sort`, it will sort for you automagically.